moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -367,6 +367,19 @@ export const PROJECT_LIST_VIEW_FILTERS_CONFIG = {
       type: "array",
       operators: ["subprojects_array_is_null", "subprojects_array_is_not_null"],
     },
+    {
+      name: "components",
+      label: "Components",
+      placeholder: "component",
+      type: "string",
+      operators: [
+        "string_contains_case_insensitive",
+        "string_begins_with_case_insensitive",
+        "string_ends_with_case_insensitive",
```

Comment:
I took out "string_equals_case_sensitive" and "string_does_not_equal_case_sensitive", let me know if they should go back in. 

I am also wondering how often someone will want to use these, string begins with or ends with?